.github/actions/image-scan/action.yml

```
@@ -113,7 +113,11 @@ runs:
 
         # PR作成ブランチの存在チェック
         git fetch origin $BRANCH_NAME
-        if [ $? -ne 0 ]; then
+        pr_branch_exist=$(echo $?)
+        # 修正対象行があるかチェック
+        cat ${{ inputs.dockerfile-path }} | grep "package-cache-v"
+        version_up_line_exist=$(echo $?)
```

ここは何かechoしてる意味がありそうですかね？  
サブシェル使うと意図していないexit codeがある場合があって、少し不安ですね

```suggestion
        version_up_line_exist="$?"
```

---

というのは書いたものの、そもそも修正対象業があるかどうかは、 `cat ${{ inputs.dockerfile-path }} | grep "package-cache-v"` の結果が空文字列かどうかで判定できそうですね

---

ありがとうございます！ 修正しました 🙏